x/operator/keeper/slash.go

```diff
@@ -18,7 +16,9 @@ import (
 )
 
 type SlashInputInfo struct {
```

Comment:
Please move this to the `operator/types` package.

Reply:
okay